content/blog/2018-11-11-colocr.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: "colocr"
+title: Co-localization analysis of image microscopy
+package_version: 0.1.0
+authors:
+  - name: Mahmoud Ahmed
+    url: https://github.com/MahShaaban
+date: 2018-11-11
+categories: blog
+topicid:
+tags:
+- r
+- package
+- onboarding
+- imaging
+- reproducibility
+---
+
+A few months ago, I wasn't sure what to expect when reading microscopy images. I looked at the accompanying graph to understand the data or the point the authors is trying to make. Often, the graph represents one or more measures of the so-called co-localization, but I couldn’t figure out how to interpret them. It turned out; reading the images is simple. Cells are simultaneously stained by two dyes (say, red and green) for two different proteins. The color turns yellow in the merged image when the two proteins localize in proximity. 
+
+A recent publication from our laboratory included microscopy images and quantification graph. The standard analysis protocol is to use [ImageJ](https://imagej.nih.gov/ij/index.html) with a specialized plug-in for the co-localization analysis such as [coloc2](https://imagej.net/Coloc_2) or [Fiji](https://fiji.sc). The images were processed one at a time; subtracting the background, selecting regions of interest and calculating the co-localization statistics. Luckily for me, I didn’t do the work myself, but it took a good chunk of my colleague precious time. Even worse, she had to repeat this whole process more than once with minor changes in the parameters each time. At the end, there was no way to make this process reproducible, and we included only the final output in an otherwise fully reproducible article.
```

Comment:
You might omit some of this "Luckily for me, I didn’t do the work myself, but it took a good chunk of my colleague precious time. Even worse, she had to repeat this whole process more than once with minor changes in the parameters each time. " and condense by saying the work was done manually and repeated with minor changes in the parameters each time.